bigbluebutton-html5/imports/ui/components/whiteboard/slide-controls/component.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.scss';
+import Button from '/imports/ui/components/button/component';
+import { callServer } from '/imports/ui/services/api/index.js';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+export default class SlideControls extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { sliderValue: 100 };
+
+    this.nextSlide = this.nextSlide.bind(this);
+    this.previousSlide = this.previousSlide.bind(this);
+    this.skipToSlide = this.skipToSlide.bind(this);
+    this.clickHandler = this.clickHandler.bind(this);
+    this.handleValuesChange = this.handleValuesChange.bind(this);
+  }
+
+  handleValuesChange(event) {
+    this.setState({ sliderValue: event.target.value });
+  }
+
+  // Change to the next slide
+  nextSlide() {
+    const currentSlideNum = this.props.currentSlideNum;
+    if (this.nextSlideAvailable()) {
+      callServer('publishSwitchToXSlideMessage', currentSlideNum + 1);
+    }
+  }
+
+  nextSlideAvailable() {
+    const currentSlideNum = this.props.currentSlideNum;
+    let nextSlideAvailable = true;
+
+    if (currentSlideNum == this.props.numberOfSlides) {
+      nextSlideAvailable = false;
+    }
+
+    return nextSlideAvailable;
+  }
+
+  // Change to the previous slide
+  previousSlide() {
+    const currentSlideNum = this.props.currentSlideNum;
+
+    if (this.prevSlideAvailable()) {
+      callServer('publishSwitchToXSlideMessage', currentSlideNum - 1);
+    }
+  }
+
+  prevSlideAvailable() {
+    const currentSlideNum = this.props.currentSlideNum;
+    let prevSlideAvailable = true;
+
+    if (currentSlideNum == 1) {
+      prevSlideAvailable = false;
+    }
+
+    return prevSlideAvailable;
+  }
+
+  // Change to a specific slide (using dropdown menu)
+  skipToSlide(event) {
+    const requestedSlideNum = event.target.value;
+    callServer('publishSwitchToXSlideMessage', requestedSlideNum);
+  }
+
+  clickHandler(buttonComponent) {
+    console.log('SlideControls::clickHandler()');
+  }
```

Comment:
I must keep an empty click handler for the fit-to-width and fit-to-screen buttons because onClick is a required property of Button.  These click handlers will be implemented eventually just not yet.  I removed it's parameter though, that was left over from testing :+1: 
```Failed prop type: Required prop `onClick` was not specified in `Button````